File: ruby/filter-content/filtering_get_items_by_taxonomy_term.rb
# Note: Filters work with codenames of the tags.
# Gets itmes tagged with one specific tag
delivery_client.items('elements.tags'.contains('kontent_ai'))

# Gets items tagged with a list of specific tags
delivery_client.items('elements.tags'.any('kontent_ai', 'cms'))

# Gets items tagged with at least one of multiple tags
delivery_client.items('elements.tags'.contains('headless', 'cms'))

File: ts/filter-content/filtering_get_items_by_taxonomy_term.ts
// Note: Filters work with codenames of the tags.
// Gets items tagged with one specific tag
const itemsByTag = await deliveryClient.items()
  .containsFilter('elements.tags', ['kontent_ai'])
  .toPromise();

// Gets items tagged with a list of specific tags
const itemsByAllTags = await deliveryClient.items()
  .allFilter('elements.tags', ['kontent_ai', 'cms'])
  .toPromise();

// Gets items tagged with at least one tag from the list
const itemsByAnyTag = await deliveryClient.items()
  .anyFilter('elements.tags', ['headless', 'cms'])
  .toPromise();